Mobile Buy SDK/Mobile Buy SDK Tests/BUYClientTestBase.m

```diff
@@ -29,7 +29,7 @@
 
 NSString * const BUYShopDomain_Placeholder = @"test_shop";
 NSString * const BUYAPIKey_Placeholder = @"channel_id";
-NSString * const BUYChannelId_Placeholder = @"api_key";
+NSString * const BUYAppId_Placeholder = @"api_key";
```

Comment:
We should fix this naming problem. `AppId`->"api_key" and `APIKiey` -> "channel_id".